packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -463,6 +465,17 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     return response.map(deserializeExecutionPayloadBody);
   }
 
+  async getBlobs(_fork: ForkName, versionedHashes: VersionedHashes): Promise<(BlobAndProof | null)[]> {
```

Comment:
if the EL does not have this api implemented yet it will always result in an error, from my observation when we implemented the client version api, this can lead to engine status switching from syncing to offline